public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -139,7 +147,12 @@ export class EhrViewComponent implements OnInit, OnDestroy {
           }
         }));
       this.subscriptions.push(this.searchText.valueChanges
-        .subscribe((query) => localStorage.setItem('searchText', query)));
+        .subscribe((query) => {
```

Comment:
making sure if the user has a space in the search bar that databrowser ignores it